deployment/injector.yml

```diff
@@ -1,6 +1,38 @@
 ---
-- hosts: injector
-  become: yes
-  roles:
-    - user-traffic-reader
-    - injector-configurator
+- hosts:
+    - hyperfoil_controller
+    - hyperfoil_agent
+  tasks:
+  - name: Install Java
```

Comment:
as root?

`become: yes`

Reply:
assuming ansible_user=root will not always be sufficient then yes. as root.